src/app_websocket.rs

```diff
@@ -86,13 +86,12 @@ impl AppWebsocket {
     ) -> Result<String> {
         let app_info = self.app_info.clone();
         self.inner
-            .on_signal(move |signal| {
-                if let Signal::App {
+            .on_signal(move |signal| match signal.clone() {
+                Signal::App {
                     cell_id,
                     zome_name: _,
                     signal: _,
-                } = signal.clone()
-                {
+                } => {
                     if app_info.cell_info.values().any(|cells| {
```

Comment:
I'm pretty sure this if condition is pointless, but we can remove that another time